Makefile

```diff
@@ -0,0 +1,8 @@
+deploy:
+	# composer install --prefer-dist --optimize-autoloader --no-dev
+	composer install
```

Comment:
Is it for dev or prod?

Reply:
I think we can have two commands:

`make deploy` or `make deploy:prod` for production
`make deploy:dev` for development

Reply:
Agreed!

Reply:
my next PR does this